qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/ClusteringRule.java

```diff
@@ -114,6 +114,7 @@
   private Gateway gateway;
   private CountDownLatch partitionLatch;
   private final Map<Integer, Leader> partitionLeader;
+  private final Map<Integer, SpringBrokerBridge> springBrokerBridge;
```

Comment:
Please let us talk about this. 

The SpringBrokerBridge is intended to be treated like a Singleton, and managed by Spring. So nobody should call its constructor.

Reply:
Sorry, my bad. Didn't realize that we are in the test code.